Tests/SwiftStdlibTests/CollectionExtensionsTests.swift

```diff
@@ -151,4 +151,9 @@ final class CollectionExtensionsTests: XCTestCase {
         XCTAssertEqual([Double]().average(), 0)
     }
 
+    func testContainsDuplicates() {
+        XCTAssert(["a", "b", "b", "c"].containsDuplicates)
+        XCTAssertFalse(["a", "b", "c", "d"].containsDuplicates)
```

Comment:
I'm missing the edge condition with an empty array.

Reply:
Okay, will be added.